extensions/ql-vscode/src/model-editor/external-api-usage-queries.ts

```diff
@@ -106,8 +106,7 @@ export async function runExternalApiQueries(
         message: update.message,
       }),
     token,
-    // We need to create a lock file, because the query is inside our own pack
-    createLockFile: true,
+    createLockFile: false,
```

Comment:
Odd, I thought I tried leaving it out and I got a failure, it must have been something else then (see https://github.com/github/vscode-codeql/pull/2731#issuecomment-1695300839). I had some troubles initially with a corrupted codeql cache.

If this is not needed, then I think we can get rid of the whole argument. It was the only use case I had for it and I was not very happy with having to add it.

Reply:
I don't think it's necessary in this case. The `createLockFileForStandardQuery` would create a lock file for a standard query, but this is not actually a standard query, it's just a query on the filesystem.

I see that `createLockFile` is always `false` now, so I'll remove the argument.